plugins/catalog-graph/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@backstage/plugin-catalog-graph",
+  "version": "0.1.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "private": true,
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "scripts": {
+    "build": "backstage-cli plugin:build",
+    "start": "backstage-cli plugin:serve --config ../../app-config.yaml",
+    "lint": "backstage-cli lint",
+    "test": "backstage-cli test",
+    "diff": "backstage-cli plugin:diff",
+    "prepack": "backstage-cli prepack",
+    "postpack": "backstage-cli postpack",
+    "clean": "backstage-cli clean"
+  },
+  "dependencies": {
+    "@backstage/catalog-client": "^0.3.18",
+    "@backstage/catalog-model": "^0.9.2",
+    "@backstage/core-components": "^0.4.1",
+    "@backstage/core-plugin-api": "^0.1.7",
+    "@backstage/plugin-catalog-react": "^0.4.5",
+    "@backstage/theme": "^0.2.10",
+    "@material-ui/core": "^4.12.2",
+    "@material-ui/icons": "^4.9.1",
+    "@material-ui/lab": "4.0.0-alpha.47",
```

Comment:
This version is different from what we use elsewhere. I had some trouble with the autocomplete that is resolved here. We could update to this version everywhere, I don't think that this bump included any breaking changes. On the other side we could just sit it out and wait for #7094 that includes the autocomplete.

Reply:
Actual, the build seems to fail because my version is to new #7196